python-frontend/src/main/java/org/sonar/plugins/python/api/LocationInFile.java

```diff
@@ -53,4 +64,28 @@ public int endLine() {
   public int endLineOffset() {
     return endLineOffset;
   }
+
+
+  public DescriptorsProtos.LocationInFile toProtobuf() {
+    return DescriptorsProtos.LocationInFile.newBuilder()
+      .setFileId(fileId)
+      .setStartLine(startLine)
+      .setStartLineOffset(startLineOffset)
+      .setEndLine(endLine)
+      .setEndLineOffset(endLineOffset)
+      .build();
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+    LocationInFile that = (LocationInFile) o;
+    return startLine == that.startLine && startLineOffset == that.startLineOffset && endLine == that.endLine && endLineOffset == that.endLineOffset && fileId.equals(that.fileId);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(fileId, startLine, startLineOffset, endLine, endLineOffset);
+  }
```

Comment:
As discussed: I think it would be better to remove these `equals` implementations that are currently only used for testing.